src/components/shared/index.js

```diff
@@ -85,6 +86,7 @@ const registerBootstrapComponents = (vue) => {
   vue.component('traction-card-header', BCardHeader)
   vue.component('traction-nav-item', BNavItem)
   vue.component('traction-nav', BNav)
+  vue.component('traction-input', BFormInput)
```

Comment:
Is there a reason to revert the input fields back to using bootstrap instead of the custom one

Reply:
@BenTopping 
`src/components/ont/runs/ONTFlowcell.vue`
```
:value="flowcellId"
:formatter="formatter"
```